corehq/apps/reports/templates/reports/project_health/partials/user_list.html

```diff
@@ -3,9 +3,13 @@
 <table class="table table-striped">
     <thead>
         <tr>
+            <th></th>
+            <th colspan="2">{% trans "Change in Forms Submitted" %}</th>
+        </tr>
```

Comment:
confirming that this is supposed to be two header rows (normally there's just one)